core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -771,23 +771,23 @@ public void testRowPattern()
     @Test
     public void testPrecedenceAndAssociativity()
     {
-        assertExpression("1 AND 2 OR 3", new LogicalBinaryExpression(LogicalBinaryExpression.Operator.OR,
-                new LogicalBinaryExpression(LogicalBinaryExpression.Operator.AND,
+        assertExpression("1 AND 2 OR 3", LogicalExpression.or(
```

Comment:
These would be easier to read if you put every argument in a separate line. But I guess it is going to be refactored soon, anyway.

Reply:
Yeah, I didn't want to change this too much to avoid conflicts with that other PR.